internal/exec/terraform_outputs.go

```diff
@@ -364,7 +367,7 @@ func getStaticRemoteStateOutput(
 
 	res, err := u.EvaluateYqExpression(atmosConfig, remoteStateSection, val)
 	if err != nil {
-		l.Fatal("Error evaluating the 'static' remote state backend output", "output", output, "component", component, "stack", stack, "error", err)
+		log.Fatal("Error evaluating the 'static' remote state backend output", "output", output, componentStr, component, stackStr, stack, "error", err)
```

Comment:
we can add "component" and "stack" as exceptions in linting or introduce global constants, but randomly defining componentStr and stackStr to get around lint warnings in logging I think makes it harder to understand.